components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/AuthenticationEndpointUtil.java

```diff
@@ -193,5 +195,17 @@ public static String customi18n(ResourceBundle resourceBundle, String key) {
             return i18nBase64(resourceBundle, key);
         }
     }
+
+    /**
+     * Build Tenant Qualified Url.
+     *
+     * @param context Context.
+     * @return Tenant Qualified Url.
+     * @throws URLBuilderException URLBuilderException.
+     */
+    public static String buildTenantQualifiedUrl(String context) throws URLBuilderException {
```

Comment:
Actually this is not just building a tenant qualified url right. Based on the mode it may or not. I think we have to rename this method

Reply:
Renamed the method accordingly.